Mirror/Runtime/NetworkManager.cs

```diff
@@ -666,23 +658,28 @@ public Transform GetStartPosition()
         {
             // first remove any dead transforms
             startPositions.RemoveAll(t => t == null);
-
-            if (playerSpawnMethod == PlayerSpawnMethod.Random && startPositions.Count > 0)
-            {
-                // try to spawn at a random start location
-                int index = Random.Range(0, startPositions.Count);
-                return startPositions[index];
-            }
-            if (playerSpawnMethod == PlayerSpawnMethod.RoundRobin && startPositions.Count > 0)
+            if (startPositions.Count > 0)
             {
-                if (s_StartPositionIndex >= startPositions.Count)
+                switch (playerSpawnMethod)
```

Comment:
There are only 2 options here,  no need for switch.   use `else if`  if you want

Reply:
Switch doesn't make it less readable, i think that it is better